### PR TITLE
Glossary of terms page

### PR DESCRIPTION
**Pull request by tekstrand**
## What
1. Adds a new `/docs/terms/` page to the site which has a list of common terms and their definitions
2. Adds [definition lists](https://docusaurus.community/knowledge/markdown/definition-lists/)
3. Fixes a markdown issue with two pages causing failed builds in some cases(contributing and legal)

## Why
We (Discord #docs) believe that it will be useful to have a standardized way of defining terms so that Meshtastic users can reliably find definitions without asking others.
Description lists are the semantic way of handling this in html https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

## Screenshots
Screenshot 2024-02-10 at 11 39 57 AM
Screenshot 2024-02-10 at 11 39 53 AM

### CODE REVIEW

File: docs/terms/index.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+Serial
+: A communication protocol used for the transmission of data between the Meshtastic device and a computer or other devices. Typically over USB or UART.
+
+SNR
+: Signal-to-Noise Ratio, a measure used in communications to quantify the level of a desired signal to the level of background noise. In Meshtastic and other wireless systems, a higher SNR indicates a clearer signal that can enhance the reliability and quality of data transmission.
+
+SWR
+: Standing Wave Ratio, a measure of the efficiency of the radio frequency (RF) power transmission from a transmitter through a transmission line to an antenna in Meshtastic devices. It indicates the ratio of the amplitude of a standing wave at maximum to the amplitude at minimum, with an ideal SWR close to 1:1, signifying that most of the power is transmitted to the antenna with minimal reflections.
+
+SX1262
```
Comment (pdxlocations) [inline docs/terms/index.mdx:106]: Not sure we need to define this.  If we do, we may want to define other radios.

File: docs/terms/index.mdx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+: The sending of sensor data or system metrics over the mesh network.
+
+Tranceiver
+: A device capable of both transmitting and receiving communications.
+
+Transmit
+: The act of sending data, such as messages or GPS locations, from one Meshtastic device to another over the LoRa network.
+
+TX
+: Short for Transmit.
```
Comment (pdxlocations) [inline docs/terms/index.mdx:119]: "Abbreviation" would be my pref.

**Comment by tekstrand** [comment]
Agree with all feedback @pdxlocations - will wait to see if anyone else has more before updating later today.

File: docs/terms/index.mdx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+: A component or add-on for a Meshtastic device, typically referring to a hardware piece such as a LoRa radio module, that provides specific functionalities like communication or GPS tracking.
+
+MQTT
+: A lightweight messaging protocol designed for small sensors and mobile devices, enabling efficient data transmission in the Meshtastic network for Internet connectivity and integration with IoT platforms. See https://en.wikipedia.org/wiki/MQTT. In Meshtastic, MQTT is used to connect a node to the internet, and can be used to connect multiple meshes to each other.
+
+Node
+: A unit within the Meshtastic network that can send, receive, and relay messages, helping to form and extend the mesh network's coverage.
+
+NRF52
+: A chipset of microcontroller made by Nordic, used by a number of devices. Lower power usage than ESP32.
```
[the remark this answers is not in the document]
Comment (tekstrand) [reply docs/terms/index.mdx:71]: `microcontroller chipset` is better IMO

File: docs/terms/index.mdx (hunk 1 above)
Comment (tekstrand) [reply docs/terms/index.mdx]: Yeah can't remember how that one was added and not the others. I don't think we need to define them here.

**Comment by tekstrand** [comment]
♻️ this is ready for re-review. I added a few links next to some of the terms that I thought could be helpful as well.

**Comment by pdxlocations** [comment]
DFU
Device Firmware Update, A state which a device is placed into for it to receive a firmware update

**Comment by tekstrand** [comment]
DFU added.

**Review by pdxlocations — APPROVED** [review]

**Pull request merged**